src/memgraph.cpp

```diff
@@ -927,6 +944,9 @@ int main(int argc, char **argv) {
       return memgraph::query::plan::CallProcedure::GetAndResetCounters();
     });
   }
+  memgraph::license::LicenseInfoSender license_info_sender(telemetry_server, run_id, machine_id,
+                                                           memgraph::license::global_license_checker.GetLicenseInfo(),
+                                                           std::chrono::seconds(5));
```

Comment:
The question is what to put here, we mentioned 24h, but something like 8h seems better to me?